docs/index.rst

```diff
@@ -89,5 +89,15 @@ JAX is Autograd_ and XLA_, brought together for high-performance numerical compu
    glossary
 
 
+The core JAX library focuses on scalable array operations.
+If you're looking to train neural networks, use flax_ and start with its documentation.
+Some associated tools are optax_ and orbax_.
+For an end-to-end transformer library built on JAX, see maxtext_.
```

Comment:
Can we capitalize these? MaxText, Flax, Orbax, Optax etc. That's how they seem to appear in their own readmes.

Reply:
Done!